### PR TITLE
Profile Responsiveness Update

### PR DESCRIPTION
**Pull request by kycodee**

### CODE REVIEW

File: src/components/Profile.tsx [hunk 1]
```diff
@@ -58,13 +58,13 @@ function Profile() {
   const [nickname, setNickname] = useState('');
   const [location, setLocation] = useState('Baton Rouge, LA');
   const [myMood, setMood] = useState("I'm Feeling Great!");
-  const [age, setAge] = useState('18-99+');
+  const [age, setAge] = useState('Add age here');
```
Comment (peytono) [inline src/components/Profile.tsx:61]: I like the default here being clear it hasn't been set yet. Should we do that for location, mood, and phone number as well?

File: src/components/Profile.tsx [hunk 2]
```diff
@@ -210,22 +214,36 @@ function Profile() {
 
   useEffect(() => {}, [nickname, myNameEditClicked, myAgeEditClicked, myNumberEditClicked, myMoodEditClicked]);
 
+
+  useEffect(() => {
+    axios
+      .get('/user')
+      .then((results: any) => {
+          // const curUser = { ...results };
+          console.log(results.userPicture, "heyyy")
```
Comment (peytono) [inline src/components/Profile.tsx]: Did you mean to leave this log here?

File: src/components/Resources.tsx [hunk 3]
```diff
@@ -28,30 +28,31 @@ function Resources() {
 
   function getTypesOfGrief() {
     axios.get('/resources/addResource').then((results: AxiosResponse) => {
-      setGriefObj(results.data.allResources);
-      setGriefStrSplit(results.data.allResources[0].split('<h3>'));
-      setGriefObjTypes(results.data.titles);
+      console.log(results.data)
```
Comment (peytono) [inline src/components/Resources.tsx]: Was this log left by accident?

File: src/components/Profile.tsx (hunk 1 above)
Comment (kycodee) [reply src/components/Profile.tsx:61]: Yes, I believe I want to do that for all of them. I'm updating it now.

File: src/components/Profile.tsx (hunk 2 above)
Comment (kycodee) [reply src/components/Profile.tsx]: No I didn't, I'm removing it now.

File: src/components/Resources.tsx (hunk 3 above)
Comment (kycodee) [reply src/components/Resources.tsx]: I meant to comment it out. Thanks.

File: src/components/Profile.tsx (hunk 1 above)
Comment (peytono) [reply src/components/Profile.tsx:61]: Nice!

**Review by peytono — APPROVED** [review]

**Pull request merged**